### DISCUSSION TITLE
Node 18 Update

### DISCUSSION THREAD

**Discussion by agvaldezc** [thebaselab/codeapp#964] [Q&A] [answered]
Hi! I just saw that the node-mobile project just release a new update with Node 18 and I was wondering if you already took a look at this and when we could possibly have that running here.

https://github.com/nodejs-mobile/nodejs-mobile/releases/tag/v18.17.1

**Comment by bummoblizard** [comment]
Looks great. Let’s see if I can get icu enabled in the build.

**Reply by casian2018** [reply]
ping

**Reply by bummoblizard** [reply]
@casian2018 Thanks for the heads up. A build with Node 18 will be available on TestFlight soon. But I haven't got icu enabled yet.

**Comment by bummoblizard** [comment]
Node 18 will be available on TestFlight shortly additionally with small icu and inspector support.

**Comment by bummoblizard** [comment, marked as answer]
Node 18 is now available in the App Store release.
